department-of-veterans-affairs/va.gov-cms
# Created the menus for first group of VISN19 systems

## Description

Map out and Create Admin and Editorial Menus for VISN-19 Systems

## QA steps

As admin.

- [ ] Visit <Manage/content/menus/VAMC system menus> and validate that menus displayed match this 
<img width="921" alt="Screen Shot 2020-06-29 at 9 02 35 AM" src="https://user-images.githubusercontent.com/56320995/86030184-fe67fb00-b9e8-11ea-9f27-5a4874f21de4.png">

- [-] Visit <Manage/Structure/content types/VAMC"all" >   <---This is not a thing.  Does "all" mean all of these below?
    - [x] /admin/structure/types/manage/health_care_local_facility
    - [-] /admin/structure/types/manage/health_care_local_health_service   
    - [x] /admin/structure/types/manage/health_care_region_page
    - [-] /admin/structure/types/manage/regional_health_care_service_des
    - [x] /admin/structure/types/manage/vamc_operating_status_and_alerts
 and validate that the content types have new Menu items (VISN-19 systems) selected as enabled (check box) to match this 
<img width="926" alt="Screen Shot 2020-06-29 at 8 53 53 AM" src="https://user-images.githubusercontent.com/56320995/86030304-248d9b00-b9e9-11ea-93ce-8e404e6c596c.png">

- [x] Visit <Sections/veterans health administration/VAMC facilities> and validate that menus displayed for VISN-19 match this 
<img width="1228" alt="Screen Shot 2020-06-29 at 9 17 30 AM" src="https://user-images.githubusercontent.com/56320995/86030721-b72e3a00-b9e9-11ea-8e1e-8130df715b8e.png">

-[ ] Visit <Manage/Content/Menus/CMS administration/Administration> and validate thet VISN 4 and VISN 19 systems are assigned as child.
<img width="1325" alt="Screen Shot 2020-06-29 at 8 52 06 AM" src="https://user-images.githubusercontent.com/56320995/86031078-3d4a8080-b9ea-11ea-853e-852dbaa00381.png">

## Definition of Done
